core/server/master/src/test/java/alluxio/master/file/meta/SimpleInodeLockListTest.java

```diff
@@ -212,4 +212,105 @@ public void doubleWriteLock() {
     mLockList.unlockLastInode();
     assertEquals(LockMode.WRITE, mLockList.getLockMode());
   }
+
+  @Test
+  public void readAfterWrite() {
+    mLockList.lockRootEdge(LockMode.READ);
+    assertEquals(LockMode.READ, mLockList.getLockMode());
+    mLockList.lockInode(mRootDir, LockMode.WRITE);
+    assertEquals(LockMode.WRITE, mLockList.getLockMode());
+    mLockList.lockEdge(mRootDir, mDirA.getName(), LockMode.READ);
+    assertEquals(LockMode.WRITE, mLockList.getLockMode());
```

Comment:
You are now asserting for this on https://github.com/Alluxio/alluxio/pull/9092

Reply:
I reverted to original behavior in that PR.